src/domain/space/space.about/dto/space.about.dto.update.ts

```diff
@@ -25,9 +25,4 @@ export class UpdateSpaceAboutInput {
   @IsOptional()
   @MaxLength(VERY_LONG_TEXT_LENGTH)
   who?: string;
-
-  @Field(() => Markdown, { nullable: true })
-  @IsOptional()
-  @MaxLength(VERY_LONG_TEXT_LENGTH)
-  when?: string;
```

Comment:
removed it's not used anywhere, there are a few more in the code but it is not in the entity